pytest/test_cli_features.py

```diff
@@ -0,0 +1,15 @@
+import os
+import json
+import glob
+from gramex.install import features
+
+
+def test_cli_features():
+    for root in glob.glob('pytest/features-*'):    
+        path = os.path.join(os.getcwd(), root)

```

Comment:
Make the path relative to the current script. We may not be running it from the pytest (or any specific) folder. Usually, I use `os.path.dirname(os.path.abspath(__file__))`